### PR TITLE
more YARD fixups and expansions

### PR DESCRIPTION
**Pull request by no-reply**
trying to make the docs look nice

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Review by jeremyf — DISMISSED** [review]

**Comment by jeremyf** [comment]
I'm not certain about the broken test, but am doing a bit of sleuthing.

**Comment by no-reply** [comment]
oh sorry. i thought i already pushed the fix for that

File: app/services/hyrax/adapters/nesting_index_adapter.rb [hunk 1]
```diff
@@ -33,19 +38,22 @@ def self.find_preservation_parent_ids_for(id:)
...
         solr_document = find_solr_document_by(id: id)
         coerce_solr_document_to_index_document(original_solr_document: solr_document, id: id)
       end
 
+      ##
       # @api public
       # @yieldparam id [String]
       # @yieldparam parent_id [Array<String>]
-      # Samvera::NestingIndexer.reindex_all!(extent: FULL_REINDEX)
-      # rubocop:disable Lint/UnusedMethodArgument
```
Comment (jeremyf) [inline app/services/hyrax/adapters/nesting_index_adapter.rb:48]: You need to keep this method in place;

The interface specs test if this method takes a block.  The way to test that interface is through parameters (e.g. `&block`)  However it doesn't use `block.call` (The original author, likely me, must've chosen `yield`)

Comment (no-reply) [reply app/services/hyrax/adapters/nesting_index_adapter.rb:48]: sorry i pushed the patch for this some time ago, but forgot to readd the rubocop.

we should probably fix the upstream test `&block` has historically been much slower than omitting it. it should be fine to test the actual `yield` in the shared examples, right?

Comment (jeremyf) [reply app/services/hyrax/adapters/nesting_index_adapter.rb:48]: The shared examples lean on method signatures/parameters.  So yes, we could change that test, however, it'll be much more expensive.

**Review by jeremyf — APPROVED** [review]

**Pull request merged**